Packs/Cybereason/Playbooks/playbook-Unisolate_Endpoint_-_Cybereason.yml

```diff
@@ -144,5 +157,8 @@ inputs:
   playbookInputQuery:
 outputs: []
 tests:
-- Cybereason Test
+- No tests (auto formatted)
```

Comment:
Looks like the SDK `format` command did not keep the existing test. It should have:
https://github.com/demisto/etc/issues/47716
https://jira-hq.paloaltonetworks.local/browse/CIAC-531

feel free to +1 my Jira issue about this, otherwise we are going to accidentally remove a lot of tests :)
regarding your PR, please bring back the test `Cybereason Test`

Reply:
done